packages/button/Button.tsx

```diff
@@ -1,33 +1,44 @@
-import { h, Component, prop } from 'skatejs';
+import { h, Component, prop, ComponentProps } from 'skatejs';
 
-import { ColorType, cssClassForColorType, css } from '../_helpers';
+import {
+  ColorType,
```

Comment:
hmm thats too much nit picking IMHO. But I see your point. We are importing a lot of stuff from `helpers` but that's what is it for.

Important is to always order `imports` in order 

- 3rd party
- local ( non package related ) - this should go away when we have proper build 
- package internal relative imports


Maybe we can extract this definitions to root `/packages/definitions.ts`

Also that `_helpers` folder is wrong, it should be separate package named `core` or `base`